Update prereqs for Fastly VCL redirect config (#4449)
magento/devdocs: guides/v2.1/cloud/cdn/fastly-vcl-wordpress.md

-  Identify the URL paths to redirect to the WordPress backend.

> -  Submit a support ticket requesting the following Fastly service configuration updates required to handle the redirects: 

Author: ```suggestion
-  Submit a support ticket requesting the following Fastly service configuration changes required to handle the redirects: 
```

> -  Submit a support ticket requesting the following Fastly service configuration changes required to handle the redirects: 

Author: ```suggestion
-  Submit a support ticket requesting the following Fastly service configuration changes required to use the custom VCL snippet for the WordPress redirects: 
```

>    -  Add the WordPress host to the Fastly backend configuration. Include the domain name for the project.

Author: ```suggestion
   -  Add the WordPress host to the Fastly backend configuration. Include the domain name for the WordPress host.
```
